internal/proxy/task_query.go

```diff
@@ -542,6 +549,10 @@ func (t *queryTask) PostExecute(ctx context.Context) error {
 	t.result.OutputFields = t.userOutputFields
 	metrics.ProxyReduceResultLatency.WithLabelValues(strconv.FormatInt(paramtable.GetNodeID(), 10), metrics.QueryLabel).Observe(float64(tr.RecordSpan().Milliseconds()))
 
+	if t.queryParams.isIterator && t.request.GetGuaranteeTimestamp() == 0 {
+		// first page for iteration, need to set up sessionTs for iterator
+		t.result.SessionTs = t.BeginTs()
+	}
```

Comment:
maybe all iterator query shall return session ts so that it's easier for sdk to implement iterator

Reply:
this is the first page for iterator, it's right for all sdks:
milvus only set up sessionTs for sdk-side iterator for the first time,
since second page, this sessinTs is set by iterator into guaranteeTs